cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/AbstractMapReduceContextBuilder.java

```diff
@@ -65,6 +67,7 @@ public BasicMapReduceContext build(MapReduceMetrics.TaskType type,
                                      String taskId,
                                      long logicalStartTime,
                                      String workflowBatch,
```

Comment:
I think ``workflowBatch`` should also be ``@Nullable``. Although please see https://github.com/caskdata/cdap/pull/2945/files#r33312959